fetch/data-urls/processing.any.js

```diff
@@ -0,0 +1,23 @@
+promise_test(() => fetch("resources/data-urls.json").then(res => res.json()).then(runDataURLTests), "Setup.");
+function runDataURLTests(tests) {
+  for(let i = 0; i < tests.length; i++) {
+    const input = tests[i][0];
+    let expectedMimeType = tests[i][1];
+    const expectedBody = expectedMimeType !== null ? tests[i][2] : null;
+    promise_test(t => {
+      if(expectedMimeType === "") {
```

Comment:
I would like to avoid such special expectations as much as possible: Can we define defaultType = "text/plain;charset=US-ASCII" and use it from the test cases?